prediction_market_agent/agents/microchain_agent/nft_treasury_game/scripts/reset_balance_anvil.py

```diff
@@ -57,6 +58,13 @@ def redistribute_nft_keys(rpc_url: str) -> None:
             )
             continue
         else:
+            min_balance_for_owner = xDai(1)
```

Comment:
When the nft key was sent to random address, it didn't have any xdai in it and transfer failed.

Reply:
> When the nft key was sent to random address, it didn't have any xdai in it and transfer failed.

I see 2 scenarios:
1. The agent sends the NFT to a wrong address, thus effectively throwing the key away, since that random address is not participating in the game.
2. It can send to the ADDRESS_ZERO, not controlled by anyone, thus 100% throwing this key away.

So in my head, this is acceptable behavior, like in a regular game, somebody does something stupid. So, if this is not happening on every game, I wouldn't force this to not occur.
Or how do you see it?